go/runtime/host/sandbox/sandbox_test.go

```diff
@@ -11,6 +11,8 @@ import (
 var envRuntimePath = os.Getenv("OASIS_TEST_RUNTIME_HOST_RUNTIME_PATH")
 
 func TestProvisionerSandbox(t *testing.T) {
+	const bwrapPath = "/usr/bin/bwrap" // Sensible systems only.
```

Comment:
I assume the same fix is required in `go/runtime/host/sgx/sgx_test.go` (currently not tested in CI).